Makefile

```diff
@@ -186,9 +186,9 @@ mock:
 	mockgen -source pkg/grpc/server/api.go -destination pkg/mock/grpc.go -package mock GrpcHandlers
 
 proto:
-	@protoc --proto_path=pkg/grpc/protobuf-schemas/proto/ --go_out=$(GOPATH)/src --plugin protoc-gen-go="$(GOPATH)/bin/protoc-gen-go" --go-vtproto_out=$(GOPATH)/src --plugin protoc-gen-go-vtproto="$(GOPATH)/bin/protoc-gen-go-vtproto" --go-vtproto_opt=features=marshal+unmarshal+size+flat_oneofs pkg/grpc/protobuf-schemas/proto/waves/*.proto
-	@protoc --proto_path=pkg/grpc/protobuf-schemas/proto/ --go_out=plugins=grpc:$(GOPATH)/src pkg/grpc/protobuf-schemas/proto/waves/node/grpc/*.proto
-	@protoc --proto_path=pkg/ride/meta/proto --go_out=plugins=grpc:$(GOPATH)/src pkg/ride/meta/proto/*.proto
+	@protoc --proto_path=pkg/grpc/protobuf-schemas/proto/ --go_out=$(GOPATH)/src --go-grpc_out=$(GOPATH)/src --go-vtproto_out=$(GOPATH)/src --plugin protoc-gen-go-vtproto="$(GOPATH)/bin/protoc-gen-go-vtproto" --go-vtproto_opt=features=marshal+unmarshal+size+flat_oneofs pkg/grpc/protobuf-schemas/proto/waves/*.proto
+	@protoc --proto_path=pkg/grpc/protobuf-schemas/proto/ --go_out=$(GOPATH)/src --go-grpc_out=require_unimplemented_servers=false:$(GOPATH)/src pkg/grpc/protobuf-schemas/proto/waves/node/grpc/*.proto
```

Comment:
Is it possible to use `*_opt` option to put `require_unimplemented_servers` like in first line for vtproto plugin?

Reply:
Nope, according to the [README.md](https://github.com/grpc/grpc-go/blob/master/cmd/protoc-gen-go-grpc/README.md) it's not possible.

Reply:
Yes, it is possible it is common way to pass options to plugin, the following command works correctly:
```
@protoc --proto_path=pkg/grpc/protobuf-schemas/proto/ --go_out=$(GOPATH)/src --go-grpc_out=$(GOPATH)/src --go-grpc_opt=require_unimplemented_servers=false pkg/grpc/protobuf-schemas/proto/waves/node/grpc/*.proto
```
https://github.com/wavesplatform/gowaves/blob/704e150bac3e9e299a2438c9538487e0dc90400d/Makefile#L190